pkg/storaged/storage.scss

```diff
@@ -500,6 +500,34 @@ td.storage-action {
     font-size: var(--pf-global--FontSize--xs);
 }
 
+.sidepanel-row:not(.pf-c-empty-state):hover {
+    cursor: pointer;
+}
+
+/* Add hover styling to ct-tables and sidepanels */
+
+.sidepanel-row:not(.pf-c-empty-state):hover,
+.storage-mounts .ct-table tbody tr:hover {
+    box-shadow: var(--pf-global--BoxShadow--sm);
```

Comment:
Is this really needed? we have tables with rows that are links and redirect in other pages in may pages, like network, machines etc. None of them have box-shadow in rows.

Reply:
I was copying PF4's table hover. But, yeah, I guess we don't do that anyway. I'll drop it.